### DISCUSSION TITLE
How do i get proper colors for themes?

### DISCUSSION THREAD

**Discussion by Aditeya** [neovide/neovide#2588] [Q&A]
I am running [tokionight.nvim](https://github.com/folke/tokyonight.nvim) colorscheme on [wezterm](https://wezfurlong.org/wezterm/) and it looks like this:

![image](https://github.com/neovide/neovide/assets/22963960/98c20e96-b26c-4572-b1d8-56212a7fe169)

and here is how it looks on neovide
 
![image](https://github.com/neovide/neovide/assets/22963960/08004859-2ed8-404e-b2f5-8cf992ae0919)

How do I make it look like the first image?

here are my neovide & colorscheme config:

<details>
<summary>colorscheme.lua</summary>

```lua
-- Colorschemes
vim.opt.termguicolors = true
vim.opt.syntax = 'on'

-- nord.nvim settings
vim.g.nord_contrast = true
vim.g.nord_borders = true
vim.g.nord_disable_background = true
vim.g.nord_cursorline_transparent = true
vim.g.nord_italic = false

local is_neovide = vim.fn.exists("g:neovide") ~= 0
if is_neovide then
  vim.g.nord_disable_background = false
  vim.g.nord_cursorline_transparent = false
else
  vim.cmd([[
    hi Normal guibg=NONE ctermbg=NONE
    hi Normal ctermbg=NONE guibg=NONE
    hi! NonText ctermbg=NONE guibg=NONE guifg=NONE ctermfg=NONE
  ]])
end

-- vim.cmd([[colorscheme nord]])
-- if vim.api.nvim_get_hl(0, { name = "@lsp.type.function" }) then
--   for _, group in ipairs(vim.fn.getcompletion("@lsp", "highlight")) do
--     vim.api.nvim_set_hl(0, group, {})
--   end
-- end

vim.cmd([[colorscheme tokyonight-night]])
```

</details>

<details>
<summary>neovide.lua</summary>

```lua
if vim.g.neovide then
    vim.g.neovide_theme = 'auto'
    vim.g.neovide_refresh_rate_idle = 5
    vim.g.neovide_refresh_rate = 60
    vim.g.neovide_no_idle = false
    vim.g.neovide_transparency = 0.9
    vim.g.neovide_floating_blur_amount_x = 2.0
    vim.g.neovide_floating_blur_amount_y = 2.0
    vim.g.neovide_hide_mouse_when_typing = true
    vim.g.neovide_scroll_animation_length = 0.1
    vim.g.neovide_cursor_animation_length = 0.1
    vim.g.neovide_profiler = false
end
```

</details>

Could you help me fix this?

**Comment by fredizzimo** [comment]
It depends on how you define correct. To me the Neovide image looks closer to what's demonstrated here https://github.com/folke/tokyonight.nvim/

But you can it to render too dark colors like Kitty if you modify the https://neovide.dev/configuration.html#text-gamma-and-contrast.

Another possibility is that you are using macOS, and in that case it might be an srgb issue see this:
* https://github.com/neovide/neovide/issues/2044
* https://github.com/neovide/neovide/issues/1102 (the last few comments)

**Reply by Aditeya** [reply]
I'm actually using Arch Linux

I suppose my WezTerm theme (also set to `tokyonight-night`)  is interfering with the Neovim theme?

I was messing with transparency and used [transparent.nvim](https://github.com/xiyaowong/transparent.nvim) to set it.
I found that with Transparency Disabled, on Neovide, the colors would show. However, when Enabled it would show black?

btw that's with the transparency enabled on Neovide.
